src/Components/AstroChat/AstroChat.tsx

```diff
@@ -90,16 +93,35 @@ export const AstroChat: React.FunctionComponent<AstroChatProps> = ({
     <Skeleton width="80%" />
   ) : (
     <>
-      <Alert
-        className="pf-v5-u-mb-md"
-        variant="info"
-        isInline
-        title="You are about to utilize Red Hat's Hybrid Cloud Console virtual assistant chat tool"
-      >
-        Please do not include any personal information or confidential information in your interaction with the virtual assistant. The tool is
-        intended to assist with general queries. Please see <a href="https://www.redhat.com/en/about/terms-use">Red Hat Terms of use</a> and{' '}
-        <a href="https://www.redhat.com/en/about/privacy-policy">Privacy Statement</a>.
-      </Alert>
+      {!alertClosed && (
+        <Alert
+          className="astro-v5-c-alert-welcome pf-v5-u-background-color-200 pf-v5-u-mb-md"
+          variant="info"
+          isInline
+          title="You are about to utilize Red Hat's Hybrid Cloud Console virtual assistant chat tool"
+          actionClose={<AlertActionCloseButton onClose={() => setAlertClosed(true)} />}
+        >
+          Please do not include any personal information or confidential information in your interaction with the virtual assistant. The tool is
+          intended to assist with general queries.
+          <div className="pf-v5-u-mt-md">
+          <Label className="pf-v5-u-mr-md pf-v5-u-px-md" onClick={() => setAlertClosed(true)} >
+            Got it
+          </Label>
```

Comment:
Looks like the linter is complaining that you're missing a level of indentation here